lib/utils.js

```diff
@@ -96,7 +96,7 @@ var Utils = module.exports = {
   cloneDeep: function(obj, fn) {
     return lodash.cloneDeep(obj, function (elem) {
       // Preserve special data-types like `fn` across clones. _.get() is used for checking up the prototype chain
-      if (!!Utils._.get(elem, 'clone') && typeof elem.clone === 'function') {return elem.clone(); }
+      if (Utils._ && Utils._.get && !!Utils._.get(elem, 'clone') && typeof elem.clone === 'function') {return elem.clone(); }
```

Comment:
Hmm, seems there must be something wrong with the version of lodash you have installed if this fails.

But just checking `elem && typeof elem.clone` should actually be enough here